app/transport/http/const.go

```diff
@@ -0,0 +1,8 @@
+package http
+
+const (
+	MsgUserCreated      = "user successfully created"
```

Comment:
I would write msgs that start from capital letter - "User successfully created" etc.

Reply:
Sure. I forgot about it. @andstepko told us to capitalize these messages.

Reply:
But what should we do with errors that come from validator?